ingestion/src/airflow_provider_openmetadata/lineage/openmetadata.py

```diff
@@ -51,6 +51,18 @@ def get_lineage_config() -> OpenMetadataLineageConfig:
             "lineage", "auth_provider_type", fallback="no-auth"
         )
         secret_key = conf.get("lineage", "secret_key", fallback=None)
+        token = conf.get("lineage", "token", fallback="okta")
+        if auth_provider_type == "okta":
+            return OpenMetadataLineageConfig.parse_obj(
+                {
+                    "airflow_service_name": airflow_service_name,
+                    "api_endpoint": api_endpoint,
+                    "auth_provider_type": auth_provider_type,
+                    "secret_key": secret_key,
+                    "token": token,
+                }
+            )
+
```

Comment:
@codingwithabhi we can't expect users to give us token, we need to generate that token just like we are doing with ingestion with okta
also add else if auth_provider_type == "google"

Reply:
for google we are not asking token, it generates that token

Reply:
IMO I don't think we need to check anything here, just pass those as key/value pairs and OpenMetadataLineageConfig should parse additional details